ui/packages/tidb-dashboard-lib/src/apps/Statement/pages/Detail/PlanDetail.tsx

```diff
@@ -84,8 +85,8 @@ function PlanDetail({ query }: IPlanDetailProps) {
     setDetailExpand((prev) => ({ ...prev, prev_query: !prev.prev_query }))
   const toggleQuery = () =>
     setDetailExpand((prev) => ({ ...prev, query: !prev.query }))
-  const togglePlan = () =>
-    setDetailExpand((prev) => ({ ...prev, plan: !prev.plan }))
+  // const togglePlan = () =>
+  //   setDetailExpand((prev) => ({ ...prev, plan: !prev.plan }))
```

Comment:
Why don't we need collapse the plan anymore?

Reply:
Communicated with the DBA, they don't need it, they want it be expanded at default, so they can see the full content as quickly as possible.